core/frontend/src/RealityDataSource.ts

```diff
@@ -50,13 +50,13 @@ export interface RealityDataSource {
    */
   getTileJson(name: string): Promise<any>;
 }
-/** @alpha */
+/** Utility functions for RealityDataSource
+ * @beta
+ */
 export namespace RealityDataSource {
-  /** Utility function to convert a RealityDataSourceKey into its string representation
-  * @alpha
-  */
+  /** Utility function to convert a RealityDataSourceKey into its string representation */
   export function keyToString(rdSourceKey: RealityDataSourceKey): string {
-    return `${rdSourceKey.provider}:${rdSourceKey.format}:${rdSourceKey.id}:${rdSourceKey.iTwinId}`;
+    return rdSourceKey.toString();
   }
   export function formatfromUrl(tilesetUrl: string): RealityDataFormat {
```

Comment:
Why define this here instead of defining `RealityDataFormat.fromUrl` instead?

Reply:
RealityDataFormat is an enum and I don't want to convert it to a class, seems logical to define this near createKeyFromUrl that use it.

Reply:
I understand why some other functions need to be here, because the types are defined in core-common but the implementations need to be defined in core-frontend.
But when that's not a factor, better to have the functionality defined directly on the type, not on some "utility" namespace in some other package.
Don't need to convert it to a class.
```
export enum RealityDataFormat {
  ThreeDTile = "ThreeDTile",
  OPC = "OPC",
}

export namespace RealityDataFormat {
  export function fromUrl(tilesetUrl: string): RealityDataFormat {
    return tilesetUrl.includes(".opc") ? RealityDataFormat.OPC : RealityDataFormat.ThreeDTile;
  }
}
```

Reply:
Move to RealityDataFormat namespace